crates/compiler/load_internal/src/file.rs

```diff
@@ -1843,8 +1948,8 @@ fn worker_task_step<'a>(
 
                         match result {
                             Ok(()) => {}
-                            Err(LoadingProblem::MsgChannelDied) => {
-                                panic!("Msg channel closed unexpectedly.")
+                            Err(LoadingProblem::ChannelProblem(problem)) => {
+                                panic!("Channel problem: {:?}", problem);
```

Comment:
future versions will  complain that you should use `{problem:?}`  I believe

Reply:
Good call, fixed!